### ISSUE TITLE
[Qusetion] Calibration datasets

### ISSUE THREAD

**Issue by cyc00518** [neuralmagic/AutoFP8#35]
Hi, thank you for your excellent work.

I'd like to ask some questions about calibrating datasets:

1. The issue https://github.com/neuralmagic/AutoFP8/issues/24 mentions using the `mgoin/ultrachat_2k` dataset for calibration. My question is: since QWEN is supposed to be a multilingual model, if we only use an English dataset for calibration, would it potentially degrade the model's capabilities in other languages?

2. My application scenario may involve both Traditional Chinese and English. Would it be better if the calibration dataset includes Traditional Chinese as well? Do you have any relevant suggestions?

3. The `activation_scheme="dynamic"` seems to not require a calibration dataset. What is the fundamental difference between this and the static scheme? Would its performance be worse?

Thank you in advance for your patient answers!

**Comment by cyc00518** [comment]
Hi neuralmagic team !

I found [amazing tutorial](<https://neuralmagic.com/blog/vllm-brings-fp8-inference-to-the-open-source-community>) on your blog. So, I think all my previous questions have been answered.

About calibration dataset, if I only use the English dataset, it seems that my ability in Traditional Chinese declines more significantly. Therefore, I translated `mgoin/ultrachat_2k` and combined it with the original one for quantization, which seems to work well. Thank you!
